common/perf-test-core/src/main/java/com/azure/perf/test/core/PerfStressProgram.java

```diff
@@ -249,7 +258,19 @@ public static void runTests(PerfTestBase<?>[] tests, boolean sync, int parallel,
                 forkJoinPool.invokeAll(operations);
 
                 forkJoinPool.awaitQuiescence(durationSeconds + 1, TimeUnit.SECONDS);
-            } else {
+            } else if (completableFuture) {
+                for (PerfTestBase<?> test : tests) {
+                    test.runAllAsyncWithCompletableFuture(endNanoTime).get();
+                }
+            } else if (executorService) {
+                for (PerfTestBase<?> test : tests) {
+                    test.runAllAsyncWithExecutorService(endNanoTime);
```

Comment:
Does anything run for this? The API being called returns a `Runnable` and since we aren't calling run the logic inside the `Runnable` never triggers

Reply:
yeah this one has issues, correcting it.